### ISSUE TITLE
Shouldn't `--discard-untrimmed` be added to `cutadapt`?

### ISSUE THREAD

**Issue by roey-angel** [benjjneb/dada2#785]
Hi,
I think, but I'm not entirely sure that `--discard-untrimmed` should be added to the `cutadapt` command. e.g. in the ITS pipeline.
Running with and without the option on my dataset I see that with `--discard-untrimmed` I get back a little fewer reads (about 2% less), which I assume are reads that for some reason lack the primer and hence should be discarded.

**Comment by benjjneb** [comment]
Good question. I had thought that we didn't want that option, because while the forward primer would always be expected on the forward reads, the reverse primer will only be detected some of the time. But on closer look at [the cutadapt docs](https://cutadapt.readthedocs.io/en/stable/guide.html), it seems like reads would be kept as long as one "adapter" (primer for us) is found...

Single reads:
```
--discard-untrimmed
Discard reads in which no adapter was found. This has the same effect as specifying --untrimmed-output /dev/null.
```

Paired reads:
```
--discard-untrimmed | one of the reads does not contain an adapter | both reads do not contain an adapter
```

**Comment by roey-angel** [comment]
Exactly, the way I understand it is that under the default option for paired ends (`--pair-filter=any`) if ONE of the reads is missing ALL adapters they would be discarded and under `--pair-filter=both` both reads should miss ALL adapters for them to be discarded. If it were enough for only one adapter to be missing for the read to be discarded then I would be loosing >95% of the reads when running the ITS pipeline because in most cases the read does not extend into the opposite primer.

The cutadapt documentation should be clearer on this

**Comment by benjjneb** [comment]
After some thinking, it is fairly common that the ITS tutorial is used on data that don't have the FWD/REV primers at the start of the reads, but need the revcom of the other primer removed from the end of reads from short amplicons. In this situation, you definitely don't want to discard-untrimmed.

I think the solution here is to add a box discussing that option and its appropriate usage to the ITS tutorial.

**Comment by benjjneb** [comment]
New comments on this issue from twitter/flogschwend:

https://twitter.com/flogschwend/status/1366384397113577477

```
Thanks for the DADA2 algorithm, it worked very nicely if correctly applied. However, 1 question remains: why is it the default to keep seq without primers during primer trimming? This inflated the diversity in our analysis (see below).
```

https://twitter.com/flogschwend/status/1366417436065599497

```
Thanks for the reply. I am just advocating to add '--discard-untrimmed' to cutadapt in the dada2 default workflow. I think this issue is not so clear in the tutorial - but it is discussed here: https://github.com/benjjneb/dada2/issues/785
```
